<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCurveChart</class>
 <widget class="QWidget" name="frmCurveChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="CurveChart" name="widget1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxValue">
      <double>150.000000000000000</double>
     </property>
     <property name="xStep">
      <double>30.000000000000000</double>
     </property>
     <property name="title">
      <string>简单曲线图示例1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CurveChart" name="widget2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>简单曲线图示例2</string>
     </property>
     <property name="pointColor">
      <color>
       <red>255</red>
       <green>107</green>
       <blue>107</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ckShowHLine">
       <property name="text">
        <string>显示横线</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckShowPoint">
       <property name="text">
        <string>显示坐标点</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckShowPointBg">
       <property name="text">
        <string>显示坐标点背景</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetData">
       <property name="text">
        <string>加载数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddData">
       <property name="text">
        <string>模拟数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearData">
       <property name="text">
        <string>清空数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CurveChart</class>
   <extends>QWidget</extends>
   <header>curvechart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
